src/gnatss/main.py

```diff
@@ -184,24 +219,33 @@ def get_transmit_times(
 def get_reply_times(
     cleaned_travel_times: pd.DataFrame,
     all_gps_solutions: pd.DataFrame,
+    rph_data: Union[pd.DataFrame, None],
```

Comment:
This is same case as `transmit_times` comments above.

Reply:
Thank you for catching this oversight 😄 . Actually the rph_data cannot be None value, and the typehint is wrong. I will update my PR with this fix